tests/topotests/static_routing_with_ebgp/test_static_routes_topo1_ebgp.py

```diff
@@ -275,7 +275,7 @@ def test_static_route_2nh_p0_tc_1_ebgp(request):
         )
         assert (
             result is not True
-        ), "Testcase {} : Failed" "Error: Routes is still present in RIB".format(
+        ), "Testcase {} : Failed" "Expected: Routes should not present in RIB \nError: Routes is still present in RIB".format(
```

Comment:
"Routes should not be present in RIB" (missing the "be") ... or go with @ton31337 's suggestion ... "Routes are"